files/group

```diff
@@ -5,6 +5,11 @@ nogroup:x:65534:
 ni:x:500:
 openvpn:x:499:
 niwscerts:x:498:
+systemd-bus-proxy:x:497:
```

Comment:
I assume that upstream has hardcoded these groups too, in which case, this is all ok.